tests/autoyast/installation.pm

```diff
@@ -118,6 +118,12 @@ sub run {
             $num_errors++;
         }
         elsif (match_has_tag('warning-pop-up')) {
+            # See poo#19978, no timeout on partition warning, hence need to click OK button to soft-fail
+            if (check_screen('warning-partition-reduced', 0)) {
```

Comment:
@coolo Thanks for the explanation. When I've said "so will match both tags" I meant that same screen matches two different needles, one is warning pop-up which checks only warning header and ok button is there and second needle uses autoyast-known-warning and newly introduced warning-partition-reduced tags. In general both existing needles should match without changes, but warning-pop-up needle does not, seems that due to font change.